main.py

```diff
@@ -11,6 +12,9 @@ def main():
         scale = scales.scale(root_note, mode_number)
         scales.print_scale(root_note, mode_number, scale)
 
+        audio.play_scale(scale, mode_number)
+        audio.input_play_again(scale, mode_number)
```

Comment:
Notice how the logic of starting the program over ("We're starting over!") is embedded in the `audio` module. That is a fundamental error: if you wanted to remove audio, you now need to seriously refactor your control flow. 

Imagine the main loop like this:

```python
while True:
  # input root_note, mode_number
  # print scale
  while True:
    # play audio
    if not io.input_yes_no("Play audio again?"):
      break
  if not io.input_yes_no("Would you like to go again?"):
    break
```